REFERENCE.md

```diff
@@ -92,7 +94,41 @@ Data type: `Any`
 
 The data for when there is no cache yet
 
-### extlib::default_content
+### `extlib::cidr_to_range`
+
+Type: Ruby 4.x API
+
+Imported by Tim 'bastelfreak' Meusel into voxpupuli/extlib because Yelp/netstdlib got abandoned
```

Comment:
are we compatible, license-wise?

Reply:
https://github.com/Yelp/puppet-netstdlib it's apache2. If I understand the license correctly, we're allowed to copy and modify it.

Reply:
so it's the same license, especially then, yes